### DISCUSSION TITLE
How to disable battery segment?

### DISCUSSION THREAD

**Discussion by victorbadaro** [JanDeDobbeleer/oh-my-posh#1600] [Troubleshoot] [answered]
I already hide it by removing it from theme file, but it still running (and taking too much time to be loaded).

**Comment by JanDeDobbeleer** [comment, marked as answer]
You only have to remove it from the theme/config but then make sure you're editing the right one.

**Reply by victorbadaro** [reply]
I see
I removed it from: `C:\Users\victor.badaro\AppData\Local\Programs\oh-my-posh\themes\jandedobbeleer.omp.json`

It's not being shown by running `oh-my-posh --debug --shell uni --config="$POSH_THEME"`. That's cool.

![image](https://user-images.githubusercontent.com/9096344/149819845-5353c52e-17e6-48c5-a490-2c04813dab1c.png)

---
But it's still being shown by running `oh-my-posh --config ~/.mytheme.omp.json --debug --shell uni`

![image](https://user-images.githubusercontent.com/9096344/149819991-b9308cd6-315d-44ce-a1b7-f9dcbdaea0aa.png)


Is that right? I really don't know. I might be confused.

**Reply by victorbadaro** [reply]
Sorry. I was referencing the wrong file on the second one.
That's right now:
![image](https://user-images.githubusercontent.com/9096344/149821148-700c3961-b54e-458d-b6b5-c7a8fc065f4b.png)

Thank you so much

**Comment by rosmur** [comment]
Just a headsup to anyone who got tripped up by this like I did: Remove the battery related items in both the theme (like atomic) and the base schema.
